app/Http/Controllers/VoteController.php

```diff
@@ -79,21 +86,15 @@ public function store(VotesRequest $request)
     public function show($id)
     {
         $vote = Vote::findOrFail($id);
+        if (Auth::user()->id &&
+            !VoteUniqueView::where(['vote_id' => $vote->id, 'user_id' => Auth::user()->id])->first()
```

Comment:
Probably better to create model method for this. Something like: isInUse($voteId, $userId)